packages/web/src/components/tipping/tip-audio/TipProfilePicture.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react'
+
+import cn from 'classnames'
+
+import { SquareSizes } from 'common/models/ImageSizes'
+import { User } from 'common/models/User'
+import { Nullable } from 'common/utils/typeUtils'
+import UserBadges from 'components/user-badges/UserBadges'
+import { useUserProfilePicture } from 'hooks/useUserProfilePicture'
+
+import styles from './TipAudio.module.css'
+
+export const TipProfilePicture = ({
+  user,
+  className = '',
+  imgClassName = ''
+}: {
+  user: Nullable<User>
```

Comment:
nit but i prefer `TipProfilePictureProps` defined above, makes it easier to export and read imo